spec/models/waste_carriers_engine/new_registration_workflow/start_form_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module WasteCarriersEngine
+  RSpec.describe NewRegistration do
+    subject(:new_registration) { build(:new_registration, start_option: start_option) }
+
+    describe "#workflow_state" do
+      context ":start_form state transitions" do
```

Comment:
Do we need a test that this cannot go `back`?

Reply:
I lean towards not testing that something isn't defined in cases like this. After all, the `next` and `back` event is not something built-in that we get by default, but something we do deliberately define. Hence, I don't think we need that test coverage.